package/kedro_viz/models/graph.py

```diff
@@ -43,6 +43,10 @@ def _strip_namespace(name: str) -> str:
     return re.sub(pattern, "", name)
 
 
+def _strip_tags(name: str) -> str:
+    return name.replace("<", "").replace(">", "")

```

Comment:
Interesting... I quite like the `<>` to indicate that these are not normal functions.

Reply:
One thing I am quite confused as to the presence of  the`<>`tag being the problem behind rendering the name as it is returned as part of a string - I think it is possible for us to keep it within the name?

Reply:
it is very weird. this is how it looks...
<img width="1222" alt="Screenshot 2022-01-18 at 09 52 20" src="https://user-images.githubusercontent.com/37628668/149913673-bc1fe252-0276-42ad-8155-c9c2fa338a4e.png">

Reply:
Same occurs with lambda.

Reply:
in the FE, we read it like this _dangerouslySetInnerHTML={{ __html: label }}_ 

maybe we can change this?

Reply:
Ahhhhhhhhhhh `_dangerouslySetInnerHTML`

Reply:
yea that explains why - not sure why we set the label this way, but if there is no use for this then we can either take the `_dangerouslySetInnerHTML` out, or set a small function on the FE to replace the `<` tag in the string with `&lt;`. ( I might say the latter might be better as it is likely this is set up this way for some reason.)

Reply:
@rashidakanchwala does it work if you do this?
```
def _strip_tags(name: str) -> str:
    return name.replace("<", "&lt;").replace(">", "&gt;")
```

Reply:
If so then we can easily just do this on the backend and avoid removing the `_dangerouslySetInnerHTML` in case it's there for a good reason.

Reply:
Hi, I had to make the above change on the FE. because if i did it in the backend, then the metadata panel was messed up. Now, it works fine
<img width="1413" alt="Screenshot 2022-01-18 at 12 29 35" src="https://user-images.githubusercontent.com/37628668/149940564-7a616c07-ac70-4b0c-bfcd-281bed4a56ee.png">